DataConnectors/Sophos Cloud Optix/Connector_REST_API_SophosCloudOptix.json

```diff
@@ -0,0 +1,109 @@
+{
+    "id": "SophosCloudOptix",
+    "title": "Sophos Cloud Optix",
+    "publisher": "Sophos",
+    "descriptionMarkdown": "The [Sophos Cloud Optix](https://www.sophos.com/products/cloud-optix.aspx) connector allows you to easily connect your Sophos Cloud Optix logs with Azure Sentinel, to view dashboards, create custom alerts, and improve investigation. This gives you more insight into your organization's cloud security and compliance posture and improves your cloud security operation capabilities.",
+    "graphQueries": [
+        {
+            "metricName": "Total data received",
+            "legend": "Sophos Cloud Optix",
+            "baseQuery": "SophosCloudOptix_CL"
+        }
+    ],
+    "sampleQueries": [
+        {
+            "description" : "Top 10 Optix alerts raised for your cloud environment(s)",
+            "query": "SophosCloudOptix_CL\n | summarize count() by alertDescription_s\n | top 10 by count_"
+        },
+        {
+            "description" : "Top 5 environments with High severity Optix alerts raised",
+            "query": "SophosCloudOptix_CL\n | where severity_s == 'HIGH'\n | summarize count() by accountId_s\n | top 2 by count_"
+        }
+    ],
+    "dataTypes": [
+        {
+            "name": "SophosCloudOptix_CL",
+            "lastDataReceivedQuery": "SophosCloudOptix_CL\n | summarize Time = max(TimeGenerated)\n | where isnotempty(Time)"
+        }
+    ],
+    "connectivityCriterias": [
+        {
+            "type": "IsConnectedQuery",
+            "value": [
+                "SophosCloudOptix_CL\n | summarize LastLogReceived = max(TimeGenerated)\n | project IsConnected = LastLogReceived > ago(30d)"
+            ]
+        }
+    ],
+    "availability": {
+        "status": 1,
+        "isPreview": true
+    },
+    "permissions": {
+        "resourceProvider": [
+            {
+                "provider": "Microsoft.OperationalInsights/workspaces",
+                "permissionsDisplayText": "read and write permissions are required.",
+                "providerDisplayName": "Workspace",
+                "scope": "Workspace",
+                "requiredPermissions": {
+                    "write": true,
+                    "read": true,
+                    "delete": true
+                }
+            },
+            {
+                "provider": "Microsoft.OperationalInsights/workspaces/sharedKeys",
+                "permissionsDisplayText": "read permissions to shared keys for the workspace are required. [See the documentation to learn more about workspace keys](https://docs.microsoft.com/azure/azure-monitor/platform/agent-windows#obtain-workspace-id-and-key)",
+                "providerDisplayName": "Keys",
+                "scope": "Workspace",
+                "requiredPermissions": {
+                    "action": true
+                }
+            }
+        ],
+        "customs": [

```

Comment:
Customs in the template is a placeholder for custom permissions. Remove "customs" permissions here if Sophos does not have any custom permissions.

Reply:
Removed the 'customs' placeholder